exercises/some/some.js

```diff
@@ -13,11 +13,23 @@
  */
 function some(collection, predicate) {
   // This is your job. :)
+  for (let item of collection) {
```

Comment:
If you have `every`, you could also do:

```javascript
function some(collection, predicate) {
  return !every(collection, item => !predicate(item));
}
```

That is, `predicate` is `true` for SOME items if it's not `false` for EVERY item.

You could also write a higher-order `not` function that gives you the negated predicate, like:

```javascript
function not(predicate) {
  return item => !predicate(item);
}
```

Then:

```javascript
function some(collection, predicate) {
  return !every(collection, not(predicate));
}
```